miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/analytics/MiniAppAnalytics.kt

```diff
@@ -17,34 +18,90 @@ private inline fun <T> whenHasAnalytics(callback: () -> T) {
 
 /** Only init when analytics dependency is provided. */
 @Suppress("SwallowedException", "TooGenericExceptionCaught")
-internal class MiniAppAnalytics(val rasProjectId: String) {
+internal class MiniAppAnalytics(
+    val rasProjectId: String,
+    private val configs: List<MiniAppAnalyticsConfig>
+) {
 
     companion object {
-        var instance: MiniAppAnalytics? = null
-
-        fun init(rasProjectId: String) = whenHasAnalytics {
-            instance = MiniAppAnalytics(rasProjectId)
+        internal fun sendAnalyticsDefault(
+            rasProjectId: String,
+            eType: Etype,
+            actype: Actype,
+            miniAppInfo: MiniAppInfo?
+        ) = try {
+            val params = createParams(
+                rasProjectId = rasProjectId,
+                acc = BuildConfig.ANALYTICS_ACC,
+                aid = BuildConfig.ANALYTICS_AID,
+                actype = actype,
+                miniAppInfo = miniAppInfo
+            )
+            whenHasAnalytics {
```

Comment:
Do we need to additionally write this check when `trackEvent` method already has the same line?

Reply:
good catch!! ,thanks @khairul-alam-licon